### ISSUE TITLE
A50 Future trading in Singapore exchange day session is not returning anything

### ISSUE THREAD

**Issue by shadow-of-life-bravo** [alpha-xone/xbbg#33]
Hi,
Thanks for your effort in maintaining this great library. When I was trying to do some intraday analysis for the A50 Future (XU1 Index).
I am retriving data in the following way:
```
blp.bdib("XU1 Index", dt="2021-03-24", session="day", ref="FuturesSingapore")
```
Where the day session trading hour is supposed to be between 9am-16:35pm as it's defined in `exch.yml`.
```
FuturesSingapore:
  tz: Asia/Singapore
  allday: [1700, 445]
  day: [900, 1635]
  night: [1700, 445]
```

Thanks!

**Comment by alpha-xone** [comment]
By default, intraday data is getting hours between `allday` and trim the `day` session afterward.

Need to change exchange info to below:

```
FuturesSingapore:
  tz: Asia/Singapore
  allday: [1700, 1635]
  day: [900, 1635]
  night: [1700, 445]
```

Will update this in the next alpha version.

**Comment by alpha-xone** [comment]
To reload data, you can do:

```
blp.bdib("XU1 Index", dt="2021-03-24", session="day", cache=False)
```

**Comment by alpha-xone** [comment]
fixed in 0.7.6a6

**Comment by shadow-of-life-bravo** [comment]
Perfect! Thanks for the prompt response. Appreciate it！
